GOVERNANCE_pme.md

```diff
@@ -0,0 +1,188 @@
+
+## Decision Making: Lazy Consensus
+
+Most decisions in PyPSA-meets-Earth are made using **lazy consensus**: if no one objects within a reasonable period (typically 5-7 days), the proposal is accepted. Only in rare cases where consensus cannot be reached, a more formal vote may be called (see "Supermajority" below).
+
+## Community Roles and Path to Maintainer
+
+Roles apply equally to code and non-code activities (e.g., documentation, community, tools, streams). The following table lists the roles within the PyPSA-meets-Earth community, describing:
+
+- General responsibilities expected by individuals in each role
+- Requirements necessary to join or remain in a role
+- Associated access rights
+
+<table>
+  <thead>
+    <tr>
+      <th>Role</th>
+      <th>Responsibilities</th>
+      <th>Requirements</th>
+      <th>Access Rights</th>
+      <th>Scope</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Participant</td>
+      <td>
+        <ul>
+          <li>Follow the <a href="https://lfprojects.org/policies/code-of-conduct/">Code of Conduct</a></li>
+          <li>Engage in discussions, submit issues and PRs, contribute to code or non-code streams</li>
+        </ul>
+      </td>
+      <td>None</td>
+      <td>
+        <ul>
+          <li>Can submit PRs and issues from forks</li>
+          <li>Can join the PyPSA-meets-Earth Discord channel (<a href="https://discord.gg/AnuJBk23FU">join here</a>)</li>
+          <li>Can take part in community discussions</li>
+        </ul>
+      </td>
+      <td>GitHub organization</td>
+    </tr>
+    <tr><td colspan="5"><em>Inherits from Participant</em></td></tr>
+    <tr>
+      <td>Contributor</td>
+      <td>
+        <ul>
+          <li>Report and help resolve issues</li>
+          <li>Answer community questions</li>
+          <li>Review and provide feedback on issues/PRs</li>
+          <li>Contribute to code, documentation, tools, or community streams</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Active participation (e.g., merged contribution, regular discussion, or content creation)</li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>Can review PRs</li>
+          <li>May close/open/reassign issues</li>
+          <li>May request PR reviews</li>
+          <li>Can mark duplicates</li>
+        </ul>
+      </td>
+      <td>Specific repo(s) or streams under pypsa-meets-earth</td>
+    </tr>
+    <tr><td colspan="5"><em>Inherits from Contributor</em></td></tr>
+    <tr>
+      <td>Maintainer</td>
+      <td>
+        <ul>
+          <li>Guide project direction and quality</li>
+          <li>Help cut releases</li>
+          <li>Mentor contributors</li>
+          <li>Review and merge PRs</li>
+          <li>Coordinate across streams (code, docs, tools, community)</li>
```

Comment:
I think we may need to agree on terminology. My understanding is that streams as PyPSA-Earth, PyPSA-Distribution, PyPSA-Earth-Status, Outreach etc. While code, docs and tools can be parts of each stream

Reply:
I agree with you, this is fixed now in f3a9c6e0ac3483edd46635efaf60fd7895a282f0